### ISSUE TITLE
Missing module map

### ISSUE THREAD

**Issue by anderscarling** [smooch/smooch-ios#114]
Smooch is not importable using `@import` syntax as the framework contains no `Modules/module.modulemap` file, the fix should be really easy though.

For an example for how to add this look at [AppsFlyerFramework](https://github.com/AppsFlyerSDK/AppsFlyerFramework/blob/master/AppsFlyerTracker.framework/Modules/module.modulemap) or a local Framework such as `/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS10.0.sdk/System/Library/Frameworks/LocalAuthentication.framework/Modules/module.modulemap`

From what I gather something like this should be enough, and it seems to work when I tried to add it locally:

```
framework module Smooch {
  umbrella header "Smooch.h"

  export *
  module * { export * }
}
```

**Comment by mspensieri** [comment]
Thanks for the report @anderscarling, we'll look into your suggestion.

For prioritization purposes, can you explain what impact this has on your application? What does `@import` provide that `#import` doesn't accomplish already?

**Comment by anderscarling** [comment]
Atm it's mostly a matter of uniformity, but another big benefit would be to enable importing the header directly into swift without using the bridging header.

I do also think it will enable some optimizations that can improve build time some, but that's probably not a big deal since it's quite a few and quite small headers.

I think Xcode should be able to generate the file for you.

Some more information here https://dev.myscript.com/forums/topic/frameworks-should-include-a-module-map-for-swift/
and http://clang.llvm.org/docs/Modules.html
and http://nsomar.com/modular-framework-creating-and-using-them/

**Comment by williamsjj** [comment]
Being able to use @import would be great. It's always the sign of an unpolished framework to me when it's missing nullability specifiers or needs a bridging header.

**Comment by wmora** [comment]
We just released [v5.5.2](https://github.com/smooch/smooch-ios/releases/tag/5.5.2) with this feature. Thanks for reporting @anderscarling
